ush/config.sh.RRFS_dev1

```diff
@@ -9,31 +9,55 @@ if [[ -n $RESERVATION ]] ; then
   PARTITION_DEFAULT=kjet
   PARTITION_FCST=kjet
   PARTITION_GRAPHICS=kjet
+  PARTITION_ANALYSIS=kjet
 fi
 
 FIXLAM_NCO_BASEDIR=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_lam.20210128
 FIXgsm=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_am.20210128
 GWD_HRRRsuite_BASEDIR=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_lam.20210128
 
+FIX_GSI=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_gsi
+FIX_CRTM=/home/rtrr/FIX_EXEC_MODULE/crtm/CRTM_v2.3.0
+
 VERBOSE="TRUE"
 
 RUN_ENVIR="nco"
 PREEXISTING_DIR_METHOD="rename"
 
 PREDEF_GRID_NAME=RRFS_CONUS_3km
 
+DO_DACYCLE="true"
+
 QUILTING="TRUE"
 CCPP_PHYS_SUITE="FV3_HRRR"
-FCST_LEN_HRS="36"
-LBC_SPEC_INTVL_HRS="6"
-
+FCST_LEN_HRS="12"
+FCST_LEN_HRS_CYCLES=(12 3 3 12 3 3 12 3 3 12 3 3 12 3 3 12 3 3 12 3 3 12 3 3)
```

Comment:
We will but current configuration cannot do it because the boundary condition from normal RAP forecast cycles is not long enough. We need to create a long forecast task group for the long forecast.